wasm/src/malloc.c

```diff
@@ -262,6 +265,7 @@ static struct heap_block * __opa_malloc_reuse_varying(struct heap_blocks *blocks
     return NULL;
 }
 
+WASM_EXPORT(opa_free) // TODO(sr): undocumented export
```

Comment:
We make references to freeing memory for the `opa_value_add_path` and `opa_value_remove_path`. Minimally, we could just add an entry in the table that lists the exports.